### ISSUE TITLE
Compatible loss functions

### ISSUE THREAD

**Issue by SebastianPopescu** [idiap/importance-sampling#1]
Is this method useful also in a regression setting? For example with the mean absolute error

**Comment by angeloskath** [comment]
You can absolutely use it in a regression setting. See A.1 at https://arxiv.org/abs/1706.00043 . Regarding the code, if you encounter problems just submit a specific issue here and we 'll try to resolve it as soon as possible.

The code does some programmatic changes of the computation graph (see https://github.com/idiap/importance-sampling/blob/master/architecture.svg) but it should work out of the box for single input single output neural networks.
